openapi/crm/crm.json

```diff
@@ -0,0 +1,319 @@
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "contacts Resource": {
+                    "value": {
+                      "CRMContact": {
+                        "title": "CRMContact",
+                        "type": "object",
+                        "properties": {
+                          "system__id": {
+                            "type": "string",
+                            "description": "The globaly unique id for this contact record"
+                          },
+                          "standard__first_name": {
+                            "type": "string",
+                            "title": "First Name",
+                            "description": "The given name of the contact"
+                          },
+                          "standard__last_name": {
+                            "type": "string",
+                            "maxLength": 50
+                          },
+                          "system__created_at": {
+                            "type": "string",
+                            "format": "date-time",
+                            "readOnly": true
+                          },
+                          "standard__email": {
+                            "type": "string",
+                            "format": "email"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "headers": {}
+          },
+          "404": {
+            "description": "User Not Found"
+          }
+        },
+        "operationId": "ListFieldSchema",
+        "description": "Retrieve field schema based on namespace."
+      }
+    },
+    "/{namespace}/{resourceTypeCode}/meta/fields/{fieldId}/options": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
```

Comment:
We normally use the plural form of the resource type name for URLs. We should probably do the same here. `contacts` = `Contact`, `companies` = `Company`, `activities` = `Activity`.

Reply:
In CRM they are using as `Contact` in the `crmObjectType`. Are you suggesting to change just the description or the `crmObjectType` as well ?